test/images/image-util.sh

```diff
@@ -46,7 +46,7 @@ build() {
   if [[ -f ${IMAGE}/BASEIMAGE ]]; then
     archs=$(listArchs)
   else
-    archs=${!QEMUARCHS[@]}
+    archs=${!QEMUARCHS[*]}
```

Comment:
```
In ./test/images/image-util.sh line 49:
    archs=${!QEMUARCHS[@]}
          ^--------------^ SC2124: Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.
```